torch/csrc/jit/tensorexpr/ir_simplifier.h

```diff
@@ -610,6 +609,8 @@ class TORCH_API IRSimplifier {
   }
 };
 
+// Flattens the buf and performs the simplifier on the flattened dims.
+const Expr* buf_flattening_helper(const Buf* v);
```

Comment:
Nit: this function might benefit from a more specific name.

Reply:
Any suggestions? :)

Reply:
Maybe `buf_flat_size` or something like that reflecting what this function is computing?